docs/whatsnew/7.0.rst

```diff
@@ -436,6 +437,65 @@ simple example might be:
     >>> stats.biweight_location(), stats.biweight_scale()  # doctest: +FLOAT_CMP
     (np.float64(67.98055399699436), np.float64(64.82889460022386))
 
+.. _whatsnew_7_0_wcsaxes_default_axes:
+
+Automatic placement of axis and tick labels for ``WCSAxes``
+===========================================================
+
+``WCSAxes`` now automatically selects which coordinates are displayed on which axes
+of a plot, a change from the previous behavior of using the ordering of the
+world axes in the WCS object.
+
+The selection is done by first placing the coordinates on the different axes in
+the order in which they appear in the WCS and then trying different permutations
+of coordinates on different axes to see if it is possible to increase the number
+of tick labels. This addresses the situation where ticks are shown along an axis
+where the world coordinate hardly varies when it varies substantially along the
+other axis.
```

Comment:
Is this description still accurate?

Reply:
Yes I edited the start of the paragraph to reflect the algorithm here so should be correct now.